### PR TITLE
finish ip_resolve option started in 6.0.0

### PR DESCRIPTION
**Pull request by chuckfloyd**
# Description

Finish ip_resolve property changes started in 6.0.0. Currently unable to render the attribute in the template.

## Issues Resolved

https://github.com/sous-chefs/yum/issues/180 - in part

## Check List

- [ ] All tests pass. See TESTING.md for details.
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 This file is used to list changes made in each version of the yum cookbook.
 
+## Unreleased
```
Comment (ramereth) [inline CHANGELOG.md:5]: Please add a new line here so the tests pass.

**Review by ramereth — CHANGES_REQUESTED** [review]

**Comment by ramereth** [comment]
@chuckfloyd I think you also need to rebase this to the master branch as you have some conflicts. Can you please do that?

File: resources/globalconfig.rb [hunk 2]
```diff
@@ -77,7 +77,7 @@
...
 property :installonlypkgs, String, description: 'List of package provides that should only ever be installed, never updated.  Kernels in particular fall into this category. Defaults to kernel, kernel-bigmem, kernel-enterprise, kernel-smp, kernel-modules, kernel-debug, kernel- unsupported, kernel-source, kernel-devel, kernel-PAE, kernel- PAE-debug.'
 property :installroot, String, description: 'Specifies an alternative installroot, relative to which all packages will be installed.'
-property :ip_resolve, Integer, equal_to: %w(4 6), description: "Determines how yum resolves host names. '4': resolve to IPv4 addresses only. '6': resolve to IPv6 addresses only."
+property :ip_resolve, String, equal_to: %w(4 6), description: "Determines how yum resolves host names. '4': resolve to IPv4 addresses only. '6': resolve to IPv6 addresses only."
```
Comment (ramereth) [inline resources/globalconfig.rb:80]: Why are you changing this to a `String`? It probably should support either that or `Integer`.

**Review by ramereth — CHANGES_REQUESTED** [review]

File: resources/globalconfig.rb (hunk 2 above)
Comment (chuckfloyd) [reply resources/globalconfig.rb:80]: The options are used as a string, it's a version string rather than an integer.

**Review by ramereth — APPROVED** [review]

**Comment by kitchen-porter** [comment]
Released as: [6.1.0](https://github.com/sous-chefs/yum/releases/tag/6.1.0)

**Pull request merged**